Remotion/Core/Core/Logging/LogExtensions.cs

```diff
@@ -116,7 +116,7 @@ public static void Log (this ILog log, LogLevel logLevel, object message)
     /// <include file='..\doc\include\Logging\LogExtensions.xml' path='LogExtensions/LogFormat/param[@name="format" or @name="args"]' />
     /// <exception cref="ArgumentOutOfRangeException">Thrown if the <paramref name="eventID"/> is outside the range of an unsigned 16-bit integer. </exception>
     [StringFormatMethod ("format")]
-    public static void LogFormat (this ILog log, LogLevel logLevel, int eventID, string format, params object[] args)
+    public static void LogFormat (this ILog log, LogLevel logLevel, int eventID, string format, params object?[] args)
```

Comment:
This file also gets an update once we are done with ILog and Log4NetLog NULL question.